main_test.go

```diff
@@ -1,8 +1,218 @@
 package main
 
 import (
+	"net/http"
+	"net/http/httptest"
 	"testing"
+
+	"github.com/sensu/sensu-go/types"
 )
 
 func TestMain(t *testing.T) {
```

Comment:
Great tests below! We can probably remove the `TestMain` func since it's not being utilized.